wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,288 @@
+
+[Detect the user's network](../how-to/manage-networks/detect-network.md) to ensure all RPC requests are submitted to the currently connected network.
+Add the following code to `src/detect.js`, which uses the [`eth_chainId`](/wallet/reference/eth_chainId)
+RPC method to detect the chain ID of the user's current network, and listens to the [`chainChanged`](/wallet/reference/provider-api/#chainchanged) provider event to detect when the user changes networks:
+
+```js title="detect.js"
+const chainId = await window.ethereum 
+  .request({ method: "eth_chainId" });
+
+window.ethereum
+  .on("chainChanged", handleChainChanged);
+
+function handleChainChanged(chainId) {
+  // We recommend reloading the page, unless you must do otherwise.
+  window.location.reload();
+}
+```
+
+### 5. Access a user's accounts
+
+To interact with Ethereum on the user's behalf, such as sending transactions or requesting balances, your dapp needs to [access the user's accounts](../how-to/connect/access-accounts.md) by calling [`eth_requestAccounts`](/wallet/reference/eth_requestaccounts).
+
+Add the following code to `src/detect.js`, which creates a button to allow users to connect to MetaMask from your dapp.
+Selecting the button activates the call to `eth_requestAccounts`, allowing you to access the user's accounts.
+
+```jsx title="detect.js"
+// You should only attempt to request the user's account in response to user interaction, such as
+// selecting a button. Otherwise, you risk spamming the user. If you fail to retrieve
+// the user's account, you should encourage the user to initiate the attempt.
+const ethereumButton = document.querySelector(".enableEthereumButton");
+const showAccount = document.querySelector(".showAccount");
+
+ethereumButton.addEventListener("click", () => {
+  getAccount();
+});
+
+// While awaiting the call to eth_requestAccounts, you should disable any buttons the user can
+// select to initiate the request. MetaMask rejects any additional requests while the first is still
+// pending.
+async function getAccount() {
+  const accounts = await window.ethereum
+    .request({ method: "eth_requestAccounts" })
+      .catch((err) => {
+        if (err.code === 4001) {
+          // EIP-1193 userRejectedRequest error.
+          // If this happens, the user rejected the connection request.
+          console.log("Please connect to MetaMask.");
+        } else {
+          console.error(err);
+        }
+      });
+  const account = accounts[0];
+  showAccount.innerHTML = account;
+}
+```
+
+Add the following HTML code to the body of `dist/index.html`, which displays the button and the current account:
+
+```html title="index.html"
+<!-- Display a connect button and the current account -->
+<button class="enableEthereumButton">Enable Ethereum</button>
+<h2>Account: <span class="showAccount"></span></h2>
+```
+
+To complete the tutorial, [run the project on a local server](#3-serve-the-project) to view the live dapp.
+
```

Comment:
Can you include one or two screenshots here portraying the final expected result?